app/controlplane/internal/usercontext/robotaccount_middleware.go

```diff
@@ -104,6 +104,13 @@ func WithAttestationContextFromRobotAccount(robotAccountUseCase *biz.RobotAccoun
 				return nil, errors.New("error retrieving the organization from the auth token")
 			}
 
+			org, err := orgUseCase.FindByID(ctx, orgID)
+			if err != nil {
+				return nil, err
```

Comment:
for the error, I´d either wrap it, or hide it altogether. About hiding it, I've been doing it in these kind authentication-related code, but I am not sure it's needed here.

In any case, pick one, and replace the naked error :)